projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt

```diff
@@ -168,6 +172,7 @@ class ProjectorServer private constructor(
     }
 
     builder.onWsOpen = { connection ->
+      clientsCount += 1
```

Comment:
Fmpov, it's too early: we should increment the counter only for the clients that have ReadyClientSettings (so they passed handshake well, including correct password and positive connection confirmation).

Reply:
Done